### ISSUE TITLE
Ending epochs for Res50？

### ISSUE THREAD

**Issue by frank-wei** [mlcommons/training_policies#371]
Looks like the doc does not document this or I missed to find it. Look at last 0.6 submission, this value varied in different submission.

**Comment by bitfort** [comment]
I believe this is captured by:
resnet	lars	lars_opt_learning_rate_decay_steps unconstrained

In the HP table.

AI - Make this clearer in the table. (This will be part of a HP name overhaul).

**Comment by ppwwyyxx** [comment]
For closed division, I'm wondering what's the reason to allow different submissions to train for different number of epochs?
If one submission managed to train for fewer epochs than others by tuning parameters such as learning rate, wouldn't that be against the spirit of "closed division"?
I saw that submitters can look at parameters of each other and adapt, but why start this game?

Thanks,
